UIKit_DZ/UIKit_DZ/UserProfileSection/UserDataViewController.swift

```diff
@@ -3,6 +3,257 @@
+        secondNameField.trailingAnchor.constraint(equalTo: view.layoutMarginsGuide.trailingAnchor).isActive = true
+        secondNameField.heightAnchor.constraint(equalToConstant: 44).isActive = true
+    }
+
+    private func makeNumberTextFieldAnchor() {
+        numberPhoneTextField.topAnchor.constraint(equalTo: secondNameField.bottomAnchor, constant: 10).isActive = true
+        numberPhoneTextField.leadingAnchor.constraint(equalTo: view.layoutMarginsGuide.leadingAnchor).isActive = true
+        numberPhoneTextField.trailingAnchor.constraint(equalTo: view.layoutMarginsGuide.trailingAnchor).isActive = true
+        numberPhoneTextField.heightAnchor.constraint(equalToConstant: 44).isActive = true
+    }
+
+    private func makeSizeShoesTextFieldAnchor() {
+        sizeShoesTextField.topAnchor.constraint(equalTo: numberPhoneTextField.bottomAnchor, constant: 10)
+            .isActive = true
+        sizeShoesTextField.leadingAnchor.constraint(equalTo: view.layoutMarginsGuide.leadingAnchor).isActive = true
+        sizeShoesTextField.trailingAnchor.constraint(equalTo: view.layoutMarginsGuide.trailingAnchor).isActive = true
+        sizeShoesTextField.heightAnchor.constraint(equalToConstant: 44).isActive = true
+    }
+
+    private func makeBirdthdayTextFieldAnchor() {
+        birdthdayTextField.topAnchor.constraint(equalTo: sizeShoesTextField.bottomAnchor, constant: 10).isActive = true
+        birdthdayTextField.leadingAnchor.constraint(equalTo: view.layoutMarginsGuide.leadingAnchor).isActive = true
+        birdthdayTextField.trailingAnchor.constraint(equalTo: view.layoutMarginsGuide.trailingAnchor).isActive = true
+        birdthdayTextField.heightAnchor.constraint(equalToConstant: 44).isActive = true
+    }
+
+    private func emailTextFieldAnchor() {
+        emailTextField.topAnchor.constraint(equalTo: birdthdayTextField.bottomAnchor, constant: 10).isActive = true
+        emailTextField.leadingAnchor.constraint(equalTo: view.layoutMarginsGuide.leadingAnchor).isActive = true
+        emailTextField.trailingAnchor.constraint(equalTo: view.layoutMarginsGuide.trailingAnchor).isActive = true
+        emailTextField.heightAnchor.constraint(equalToConstant: 44).isActive = true
+    }
+
+    private func saveConfigUserButtonAnchor() {
+        saveConfigUserButton.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: -114).isActive = true
+        saveConfigUserButton.leadingAnchor.constraint(equalTo: view.layoutMarginsGuide.leadingAnchor).isActive = true
+        saveConfigUserButton.trailingAnchor.constraint(equalTo: view.layoutMarginsGuide.trailingAnchor).isActive = true
+    }
+}
+
+// MARK: - UITextFieldDelegate
+
+extension UserDataViewController: UITextFieldDelegate {
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+        let nextTag = textField.tag + 1
+
+        if let nextResponder = textField.superview?.viewWithTag(nextTag) {
```

Comment:
У тебя же этот textfield лежит на view контроллера.
```suggestion
        if let nextResponder = view.viewWithTag(nextTag) {
```